_includes/integrations/react-quick-start.md

```diff
@@ -72,37 +72,37 @@ This procedure requires:
 
     export default App;
     ```
-    This JavaScript file will create the class `App` containing a TinyMCE editor configured to replicate the example on the [Basic example page]({{site.baseurl}}/demo/basic-example/).
-6. Provide access to TinyMCE using Tiny Cloud or by self-hosting TinyMCE.
+    This JavaScript file will create the class `App` containing a {{site.productname}} editor configured to replicate the example on the [Basic example page]({{site.baseurl}}/demo/basic-example/).
+6. Provide access to {{site.productname}} using {{site.cloudname}} or by self-hosting {{site.productname}}.
 
-    * **Tiny Cloud**
+    * **{{site.cloudname}}**
 
-        Include the `apiKey` option in the editor element and include your [TinyMCE API key]({{site.shared_baseurl}}/signup/).
+        Include the `apiKey` option in the editor element and include your [{{site.accountpage}} API key]({{site.shared_baseurl}}/signup/).
```

Comment:
This looks like the wrong variable? I think it should be `productname` instead `accountpage` (or at least `Tiny Account API Key` sounds incorrect).

Reply:
fair, changed to `cloudname`